cypress/integration/login.js

```diff
@@ -20,5 +20,12 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('TestCon User 6')
         cy.get('.main-nav').find('li').should('have.length', 2)
+
+        let d = new Date();
+        let current_day = d.getDate();
+
+        cy.get('.calendar--today').should('contain', current_day)
+
```

Comment:
please keep the style consistent, if you want to leave an empty line after a test case, make the same changes for previous test cases as well.

Reply:
Thanks for your review, not sure if I got what you have said exactly but I tried to update my code the way as I understood it. ^^